base.py

```diff
@@ -190,9 +191,22 @@ def __init__(self, addr, username, password, create_test_user=True,
     def __del__(self):
         self.browser.quit()
 
+    def _cleanup_string(self, string):
+        tags = re.compile('<.*?>')
```

Comment:
You compile the regular expression every time you call `_cleanup_string`, so it's redundant. Move the compilation to `__init__`.